examples/6. Symbolics/6. Symbolics.jl

```diff
@@ -6,10 +6,18 @@
 
 # ## Setup
 
+using Quaternions
 using RigidBodyDynamics
 using StaticArrays
 using Symbolics
 
+#-
+
+## Type piracy needed in order to make Symbolics.jl types interact well with Quaternions.jl.
+## This should be avoided — see https://docs.julialang.org/en/v1/manual/style-guide/#avoid-type-piracy.
+function Base.:/(q::Quaternions.Quaternion, x::Symbolics.Num)
+    return Quaternions.Quaternion(q.s / x.val, q.v1 / x.val, q.v2 / x.val, q.v3 / x.val)
+end
```

Comment:
I took this from https://github.com/dionysos-dev/Dionysos.jl/blob/master/BipedRobot/src/piracy.jl to check if it would fix the issue in https://github.com/JuliaRobotics/RigidBodyDynamics.jl/issues/630 and it did! So, we can close that issue when we merge this PR.